src/ray/rpc/grpc_server.cc

```diff
@@ -148,7 +148,7 @@ void GrpcServer::Run() {
       if (entry->GetMaxActiveRPCs() != -1) {
         buffer_size = entry->GetMaxActiveRPCs();
       }
-      for (int j = 0; j < buffer_size; j++) {
+      for (int j = 0; j < (buffer_size / num_threads_); j++) {
```

Comment:
Sorry, I missed this one. But I don't think it's going to be a big issue here since it should be always greater than 0.
But yeah, we should enforce it by adding CHECK.